content/reference/endpoints/change_overs.md

```diff
@@ -0,0 +1,43 @@
+# ChangeOvers
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | ChangeOver's id.
+-----------------|---------|------------|------------
+map              | String  | Read       | ChangeOver's map of booking status. There is a total of 1096 characters, each representing one day. List of statuses described in [enums section](/reference/enums#change_over-map-statuses).
```

Comment:
It's not really a booking status, more like, hmm, a map of possible actions for a given day?